### ISSUE TITLE
Add support for multiple payment trigger URLs

### ISSUE THREAD

**Issue by Klakurka** [PayButton/paybutton-server#824]
Allowing at least 2 would be nice. Perhaps we can do this with tabs in the UI?

**Comment by johnkuney** [comment]
Would these have their own post data as well?

**Comment by Klakurka** [comment]
Yeah so the idea is to switch that part of the UI to use tabs where they could have a tab for each post data + url combo.
